src/com/palmergames/bukkit/towny/db/SQLSchema.java

```diff
@@ -0,0 +1,388 @@
+package com.palmergames.bukkit.towny.db;
+
+import com.palmergames.bukkit.towny.TownyMessaging;
+import com.palmergames.bukkit.towny.TownySettings;
+import com.palmergames.bukkit.towny.db.TownySQLSource.TownyDBTableType;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author ElgarL
+ */
+public class SQLSchema {
+
+	private static final String SQLDB_NAME = TownySettings.getSQLDBName();
+	private static final String TABLE_PREFIX = TownySettings.getSQLTablePrefix().toUpperCase();

```

Comment:
toUpperCase(Locale.ROOT) might be better here to avoid weird default locale related issues.